### PR TITLE
feat: Release notes for Codacy Cloud August 2024

### PR DESCRIPTION
**Pull request by DMarinhoCodacy**
### :construction: To do
-   [x] If relevant, include the Jira issue key at the end of the pull request title
-   [x] Perform a self-review of the changes
-   [x] Fix any issues reported by the CI/CD

### CODE REVIEW

**Review by joanasteodoro — APPROVED** [review]

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+These release notes are for the Codacy Cloud updates during August 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+-  Semgrep now supports PL/SQL rules from Checkmarx (TCE-942)
+-  Radon now calculates the average complexity of each file instead of calculating the highest complexity value of the file, which created some confusion (TCE-1091)
+-  Show status checks protection rule value everywhere in the UI (ALA-1160)
+-  We made several improvements on the code patterns page (ALA-1080)
+-  Now we support multiple coding standards (ALA-1119)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:21]: Suggestions: Multiple coding standards supported: apply more than one coding standards to a single repository.

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+These release notes are for the Codacy Cloud updates during August 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+-  Semgrep now supports PL/SQL rules from Checkmarx (TCE-942)
+-  Radon now calculates the average complexity of each file instead of calculating the highest complexity value of the file, which created some confusion (TCE-1091)
+-  Show status checks protection rule value everywhere in the UI (ALA-1160)
+-  We made several improvements on the code patterns page (ALA-1080)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:20]: Suggestion: Several improvements made on the repository code patterns page

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+# Cloud August 2024
+
+These release notes are for the Codacy Cloud updates during August 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+-  Semgrep now supports PL/SQL rules from Checkmarx (TCE-942)
+-  Radon now calculates the average complexity of each file instead of calculating the highest complexity value of the file, which created some confusion (TCE-1091)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:18]: Suggestions: Radon now calculates the average complexity of each file

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+-  Show status checks protection rule value everywhere in the UI (ALA-1160)
+-  We made several improvements on the code patterns page (ALA-1080)
+-  Now we support multiple coding standards (ALA-1119)
+-  Allow configuring repo-level Code Patterns while following a Coding Standard (ALA-1120)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:22]: allowing configurations in repositories code patterns, while following a Coding Standard.

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+-  If you access the Issues page, you can see the issues grouped by patterns (ALA-1147)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:23]: Issues now grouped by patterns in Issues page (ALA-1147)

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+-  Now we support poetry.lock files when doing dependency checks with Trivy (TCE-1005)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:24]: - Support of poetry.lock files when doing dependency checks with Trivy (TCE-1005)

File: docs/release-notes/cloud/cloud-2024-08.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+## Bug fixes
+-  We fixed a problem with Sonar C# analysing non .cs files creating false positives (IO-1156)
```
Comment (RaquelHipolito) [inline docs/release-notes/cloud/cloud-2024-08.md:27]: Fixed a problem with Sonar C# analysing non .cs files creating false positives

Comment (RaquelHipolito) [reply docs/release-notes/cloud/cloud-2024-08.md:27]: Suggest to remove all "We" on the remaining bugs below

**Review by RaquelHipolito — COMMENTED** [review]
@DMarinhoCodacy Suggesting some language changes.

**Review by joanasteodoro — APPROVED** [review]

**Review by RaquelHipolito — APPROVED** [review]
@DMarinhoCodacy looks good to me

**Pull request merged**